components/desktop/freerdp/Makefile

```diff
@@ -44,8 +44,10 @@ CMAKE_OPTIONS += -DWITH_ALSA=OFF
 CMAKE_OPTIONS += -DWITH_CAIRO=ON
```

Comment:
You'll need to set or increment COMPONENT_REVISION if you are making changes like this.
Simple rule: with every PR you'll need to either change the COMPONENT_VERSION or COMPONENT_REVISION. Otherwise nobody will gets a package updated!